src/account/account-blocked.event.ts

```diff
@@ -0,0 +1,20 @@
+import type { Account } from 'account/account.entity';
+
+export class AccountBlockedEvent {
+    constructor(
+        private readonly actor: Account,
+        private readonly blocked: Account,
+    ) {}
+
+    getActor(): Account {
```

Comment:
I don't think this is a good name given Actor means something specific in ActivityPub

Reply:
`getBlocker()`? It's a bit anti-dyslexic though 🙃

Reply:
heh, i though I'd get some resistance on the naming of that 😅 My rationale was `this is the account that did the action`, but i get why this isn't ideal. Looking at `account.followed` we use `getAccount` to represent the account being followed, and `getFollower` to represent the account doing the follow. Maybe we do the same here? `getAccount` to represent the account being blocked and `getBlocker` to represent the account doing the block?